src/lib.rs

```diff
@@ -246,24 +250,29 @@ impl<'a> FMatcher<'a> {
                 (Some(x), None) => {
                     if x.trim() == WILDCARD {
                         while let Some(ptnl) = ptn_lines.next() {
+                            ptn_lines_off += 1;
                             if !self.match_line(&mut names, ptnl, "") {
                                 return Err(FMatchError {
                                     ptn: self.ptn.to_owned(),
                                     text: text.to_owned(),
+                                    ptn_line_off: ptn_lines_off,
                                     text_line_off: text_lines_off,
                                 });
                             }
                         }
                         return Ok(());
                     } else {
-                        if self.skip_blank_lines(&mut ptn_lines, Some(x)).0.is_none() {
-                            return Ok(());
+                        match self.skip_blank_lines(&mut ptn_lines, Some(x)) {
+                            (Some(_), skipped) => {
+                                return Err(FMatchError {
```

Comment:
I was supposed to do this myself, but I think the code to make the error is long enough and appears frequently enough, that we should add a `::new()` method to make things more concise. What do you think?

Reply:
I'm neutral. Can be done as a separate PR?

Reply:
Sure.